sopel_waifu/waifu.json

```diff
@@ -1453,37 +1506,112 @@
         "Yuika Mitsumine"
     ],
     "The Tatami Galaxy": [
-        "Akashi"
+        "Akashi",
+        "Hanuki Ryouko",
+        "Higuchi Keiko",
+        "Kaori"
     ],
     "To Aru Kagaku no Railgun": [
-        "Shirai Kuroko",
         "Saten Ruiko",
-        "Uiharu Kazari"
+        "Kiyama Harumi",
+        "Konori Mii",
+        "Juufuku Miho",
+        "Tessou Tsuduri",
+        "Wannai Kinuho",
+        "Awatsuki Maaya",
+        "Haruue Erii",
+        "Therestina Kihara Lifeline",
+        "Edasaki Banri",
+        "Kounoe",
+        "Hazamaya Kana",
+        "Nunotaba Shinobu",
+        "Frenda Seivelun",
+        "Mugino Shizuri",
+        "Takitsubo Rikou",
+        "Shokuhou Misaki",
+        "Kinuhata Saiai",
+        "Minori",
+        "Febri",
+        "Hokaze Junko",
+        "MISAKA 1"
+        "Kumokawa Seria",
+        "Xochitl",
+        "Taroumaru Seike",
+        "Yakumaru Itsuki",
+        "Iizumi Rita",
+        "Sakuragi Naruha",
+        "Dolly",
+        "Kouzaku Mitori",
+        "Kuriba Ryouko",
+        "Yumiya Rakko"
     ],
     "To Aru Majutsu no Index": [
-        "Index",
-        "Last Order",
+        "Index Librorum Prohibitorum",
         "Misaka \"Biribiri\" Mikoto",
-        "Misaka Imouto",
+        "Kanzaki Kaori",
         "Tsukuyomi Komoe",
+        "Kuroko Shirai",
         "Himegami Aisa",
+        "MISAKA 10032",
+        "Tsuchimikado Maika",
+        "Sasha Kreuzhev",
+        "Last Order",
+        "Kazakiri Hyouka",
+        "Itsuwa",
+        "Uiharu Kazari",
+        "Sherry Cromwell",
+        "Yomikawa Aiho",
+        "Yoshikawa Kikyou",
+        "Ryoukan",
+        "Laura Stewart",
+        "Kamijou Shiina",
+        "MISAKA 10031",
+        "MISAKA 19090",
         "Fukiyose Seiri",
-        "Mitsuko Kongo"
+        "Mitsuko Kongo",
+        "Vento of the Front",
+        "Oriana Thomson",
+        "Lidvia Lorenzetti",
+        "Lucia",
+        "Angelene",
+        "Agnese Sanctis",
+        "Orsola Aquinas",
+        "Misaka Misuzu",
+        "Musujime Awaki",
+        "Vasilisa",
+        "Uragami"
     ],
```

Comment:
This needs some rethinking. It's mind-breakingly confusing to have the **main characters** of _Railgun_ listed only under _Index_. I'm not sure listing different "MISAKA" clones is useful for this purpose; there are enough characters in the _To Aru_ universe that we probably don't need every "little sister" or minor characters with no surname.

I don't want to unilaterally decree this, though. Counterpoint(s)?

Reply:
I don't know all of the series as well as someone else invested into them so I can't say which minor supporting characters are worthy of waifu status. To be honest I don't want to add every minor character, but then later on **someone**'s going to ask "wHY iSnT mY WAifU In THe LiST???"

Reply:
As for the main characters of Railgun being listed only under Index, I'm not sure how that should work since they do wind up in the main Index series sequels.

Reply:
Regarding "wHY iSnT mY WAifU In THe LiST???", I wouldn't worry about it for minor characters. I doubt anyone's waifu is one of those girls with only a given name.

One solution would be to duplicate the main characters of _Railgun_ under that, and accept the slight increase in draw probability. But I'm also not hard against combining _the whole franchise_ into something like "To Aru Series", so we needn't deal with what looks like the "wrong" title next to people like Mikoto.

Unfortunately I've only watched part of _Railgun_, nowhere near enough of the whole _Index_ franchise to be considered "invested" or to know at a glance who's really important. Ultimately I think we should just pick a direction, and accept that it's OK to change it later.

Reply:
We're punting the larger structure issue down the road.